modules/ROOT/pages/as2-encoding.adoc

```diff
@@ -0,0 +1,23 @@
+= Configuring Receive Settings for AS2 Inbound Messages
+
+The Global AS2 encoding setting determines the character encoding type for all messages received from this partner.
```

Comment:
Please remove "Global", as it is a partner specific configuration. 
Note: Just realized that the label does state "Global". We will change it to a more meaningful value in the next release. 2.20 will still go out with "Global" in the label.